laterpay/application/Core/Bootstrap.php

```diff
@@ -159,6 +159,9 @@ private function register_admin_actions() {
         $controller = self::get_controller( 'Admin_Advanced' );
         laterpay_event_dispatcher()->add_subscriber( $controller );
 
+        $controller = self::get_controller( 'Admin_TinyMCE' );
+        laterpay_event_dispatcher()->add_subscriber( $controller );
```

Comment:
```suggestion
        laterpay_event_dispatcher()->add_subscriber( self::get_controller( 'Admin_TinyMCE' ) );
```
Could we remove usage of `$controller` and directly add the controller?